2-collectors/scala-stream-collector/project/Dependencies.scala

```diff
@@ -34,6 +34,7 @@ object Dependencies {
     val yodaTime         = "2.1"
     val yodaConvert      = "1.2"
     val kafka            = "0.10.1.0"
+    val nsqClient       = "1.0.0.RC4"
```

Comment:
is there a release incoming? I'm uneasy at the idea of having an RC as dependency.